distribution/lib/Standard/Base/0.0.0-dev/src/Data/XML.enso

```diff
@@ -25,19 +25,18 @@ from project.Data.Range.Extensions import all
 from project.Data.Text.Extensions import all
 from project.Metadata import Display, make_single_choice, Widget
 
-polyglot java import java.io.StringReader
 polyglot java import java.lang.Exception as JException
-polyglot java import javax.xml.parsers.DocumentBuilder
-polyglot java import javax.xml.parsers.DocumentBuilderFactory
```

Comment:
I am removing these `polyglot java import` statements in favor of doing more work in Java, @AdRiley, @radeusgd. This is aligned with the [plan to use static methods more](https://github.com/enso-org/enso/pull/9997#discussion_r1606033800). Please keep that in mind when introducing new `polyglot java import` (especially from JDK classes) - it has its cost!

Reply:
With 0695d1b5be the cost of `polyglot java import` of Java `interface` is even higher - it is automatically registered for _proxy creation_ - another reason to **expose only static methods classes** and not `org.w3c.dom.Node`, `org.w3c.dom.Element`, `Attr`, etc.

Reply:
- reported as #10325